content/id/docs/concepts/services-networking/service.md

```diff
@@ -0,0 +1,1065 @@
+    targetPort: 9377
+```
+
+Perhatikan bahwa penamaan _port_ hanya boleh terdiri dari karakter _alphanumeric_ _lowercase_ 
+dan _-_, serta harus dimulai dan diakhiri dengan karakter _alphanumeric_, misalnya saja `123-abc` dan `web` 
+merupakan penamaan yang valid, tapi `123_abc` dan `-web` bukan merupakan penamaan yang valid.
+
+## Memilih sendiri alamat IP yang kamu inginkan
+
+Kamu dapat memberikan spesifikasi alamat _cluster IP_ yang kamu inginkan 
+sebagai bagian dari _request_ pembuatan objek `Service`. Untuk melakukan hal ini, 
+kamu harus mengisi _fields_ `.spec.clusterIP` field. Contoh penggunaannya adalah sebagai berikut, 
+misalnya saja kamu sudah memiliki _entry_ DNS yang ingin kamu gunakan kembali, 
+atau sebuah sistem _legacy_ yang sudah diatur pada alamat IP spesifik 
+dan sulit untuk diubah. Alamat IP yang ingin digunakan pengguna haruslah merupakan alamat IP 
+yang valid dan berada di dalam _range_ _CIDR_ `service-cluster-ip-range`_ yang dispesifikasikan di dalam 
+penanda yang diberikan _apiserver_. Jika _value_ yang diberikan tidak valid, _apiserver_ akan 
+mengembalikan _response_ _code_ HTTP _422_ yang mengindikasikan _value_ yang diberikan tidak valid.
+
+### Mengapa tidak menggunakan DNS _round-robin_?
+
+Pertanyaan yang selalu muncul adalah kenapa kita menggunakan IP virtual dan bukan 
+DNS _round-robin_ standar? Terdapat beberapa alasan dibalik semua itu:
+
+   * Terdapat sejarah panjang dimana _library_ DNS tidak mengikuti _TTL_ DNS dan 
+     melakukan _caching_ hasil dari _lookup_ yang dilakukan.
+   * Banyak aplikasi yang melakukan _lookup_ DNS hanya sekali dan kemudian melakukan _cache_ hasil yang diperoleh. 
+   * Bahkan apabila aplikasi dan _library_ melakukan resolusi ulang yang _proper_, _load_ dari setiap 
+     klien yang melakukan resolusi ulang DNS akan sulit untuk di _manage_. 
+
+Kami berusaha untuk mengurangi ketertarikan pengguna untuk melakukan yang mungkin akan menyusahkan pengguna. 
+Dengan demikian, apabila terdapat justifikasi yang cukup kuat, kami mungkin saja memberikan implementasi 
+alternatif yang ada.
+
+## _Discovering services_
+
+Kubernetes mendukung 2 buah mode primer untuk melakukan `Service` - variabel _environment_ dan DNS.
+
+### Variabel _Environment_
+
+Ketika sebuah `Pod` dijalankan pada *node*, _kubelet_ menambahkan seperangkat variabel _environment_ 
+untuk setiap `Service` yang aktif. _Environment_ yang didukung adalah [Docker links compatible](https://docs.docker.com/userguide/dockerlinks/) variabel (perhatikan 
+[makeLinkVariables](http://releases.k8s.io/{{< param "githubbranch" >}}/pkg/kubelet/envvars/envvars.go#L49)) 
+dan variabel `{SVCNAME}_SERVICE_HOST` dan `{SVCNAME}_SERVICE_PORT`, dinama nama `Service` akan diubah 
+menjadi huruf kapital dan tanda _minus_  akan diubah menjadi _underscore_.
+
+Sebagai contoh, `Service` `"redis-master"` yang mengekspos _port_ TCP 6379 serta _alamat_
+_cluster IP_ _10.0.0.11_ akan memiliki _environment_ sebagai berikut:
+
+```shell
```

Comment:
below codeblock format is broken https://deploy-preview-14117--kubernetes-io-master-staging.netlify.com/id/docs/concepts/services-networking/service/#variabel-environment

probably you have remove backticks inside the codeblock